packages/amplify-cli-core/package.json

```diff
@@ -26,6 +26,7 @@
   },
   "dependencies": {
     "@aws-amplify/graphql-transformer-interfaces": "2.1.0-beta.2",
+    "@yarnpkg/lockfile": "^1.1.0",
```

Comment:
does this handle both yarn 1 and 2 ?

Reply:
only yarn 1 . I will check for yarn2 if it is supported by our projects

Reply:
added yarn2 parser since amplify project was working with yarn2